subsys/bluetooth/mesh/adv_ext.c

```diff
@@ -267,14 +265,8 @@ static void send_pending_adv(struct k_work *work)
 
 			ext_adv->adv = NULL;
 		}
-
-		(void)schedule_send(ext_adv);
```

Comment:
Did you run this change over your advertiser suspend test? If I remember correctly, this rescheduling is triggered when you suspend mesh in start callback.

Reply:
Yes, it passed.

Reply:
Ok.